bokehjs/src/lib/models/tools/gestures/range_tool.ts

```diff
@@ -122,10 +122,35 @@ export class RangeTool extends Tool {
 
   update_ranges_from_overlay(): void {
     const {left, right, top, bottom} = this.overlay
-    if (this.x_range != null && this.x_interaction)
-      this.x_range.setv({start: left, end: right})
-    if (this.y_range != null && this.y_interaction)
-      this.y_range.setv({start: bottom, end: top})
+
+    function adjustRange(value1: number | null, value2: number | null, bounds: "auto" | [number | null, number | null] | null): [number | null, number | null] {
```

Comment:
Though this is not enforced due to technical difficulty, we use snake-case for variable and function names. The exception is code referring to the standard library and external libraries, where we have no choice.